pallets/parachain-staking/tmp.md

```diff
@@ -0,0 +1,5 @@
+# TODO
+
+- delete Total storage item which tracks total staked
+  - would require that we write tools to calculate it manually from state (but this can be easily done)
+  - this would significantly simplify a lot of the new code
```

Comment:
I'll delete this, accidental commit.

We need `Total` for the issuance computation so it's not a choice to remove it unless we change the inputs for round issuance.